### DISCUSSION TITLE
how to update the latest version of stirling-pdf docker on the NAS

### DISCUSSION THREAD

**Discussion by jasli2024** [Stirling-Tools/Stirling-PDF#1653] [Q&A] [answered]
Hi,
I have installed the stirling-pdf docker on the Synology NAS, how can I updated the version if the latest version is available.

The current version of stirling-pdf app is 0.26.1 and the 0.27.0 is available.

**Comment by richie80** [comment, marked as answer]
You can use watchtower

**Reply by jasli2024** [reply]
Thanks, will check how to install watchtower.

**Reply by richie80** [reply]
https://youtu.be/mSSlrRgSAP4?si=wBNna8-N6C6MwWQq

**Reply by jasli2024** [reply]
good, thanks
